Source/WebCore/testing/Internals.cpp

```diff
@@ -2228,6 +2229,23 @@ auto Internals::lastAutoFillButtonType(const HTMLInputElement& element) -> AutoF
     return toInternalsAutoFillButtonType(element.lastAutoFillButtonType());
 }
 
+Vector<String> Internals::recentSearches(const HTMLInputElement& element)
+{
+    if (!element.isSearchField())
```

Comment:
Seems like we should check the type of the renderer rather than the type of the element.

Reply:
Sure, will change.